Cpp/odin-fg/native.jwt.cpp

```diff
@@ -51,3 +53,16 @@ const fg::frame::builtin odin::lib::jwt_payload =
         }
     };
 
+
+const fg::frame::builtin odin::lib::mint_reset_password_jwt =
+    [](fg::frame &stack, fg::json::const_iterator pos, fg::json::const_iterator end) {
+        auto username = stack.resolve_string(stack.argument("username", pos, end));
+        auto cnx = connect(stack);
+        if ( !odin::does_user_exist(cnx, username) )
+            throw fostlib::exceptions::not_implemented(__func__,
+                "The user does not appear in the database so no JWT can be minted");
+        auto token = odin::mint_reset_password_jwt(username).token();
+        fostlib::log::warning(c_odin_fg, fg::json(std::move(token)));
+        return fg::json(std::move(token));
```

Comment:
This is going to blow up. Once you've moved from something it's dead and can't be used again. Probably it's OK to assign a new value into it, but even that isn't guaranteed to work I believe. In any case, you can't move from something twice.